README.md

```diff
@@ -1,13 +1,22 @@
-# Project Name
+# Chatbot
 
 Replace this readme with your own information about your project.
 
 Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
-
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
-## View it live
+We had problems with understanding how javascript worked at first. We had a tough time even starting out the code, and it took us along time to actually understand what we did and how to write in javascript.
+
+We approached our problems with an open mind and we asked for help from our group. They came in clutch, they are great! We wanted to fix our problems on our own, as we both felt that we could learn a lot by just trying out stuff. But when we didn't get enywhere, we asked for help and it was very useful. 
```

Comment:
The lion team is a resource and we should all feel comfortable in asking for support and help. We all feel the same way!